modules/core/RapidSystem.js

```diff
@@ -95,6 +95,18 @@ export class RapidSystem extends AbstractSystem {
           label: l10n.t('rapid_feature_toggle.msBuildings.label'),
           license_markdown: l10n.t('rapid_feature_toggle.msBuildings.license_markdown')
         });
+
+        this._datasets.set('rapidMapFeatures', {
```

Comment:
This code is where the individual layers in the Rapid button are defined. Note that the 'added: false' directive would prevent this one from appearing in the list.